From 21e0356e18f7488c1803e2340000b02494d366c8 Mon Sep 17 00:00:00 2001
From: Lee Shombert <shombert@google.com>
Date: Mon, 18 Aug 2025 08:55:50 -0700
Subject: [PATCH] Clean up sqlite flags in 24Q3

This removes the sqlite trunk-stable flags from the build/release
directories in 24Q3.  These directories are used in 25Q2 and later but
not in 24Q3.  The flag values are removed to avoid confusion.

Tested by booting an image on cuttlefish and verifying that the
sqlite version ws 3.44.5.

Flag: RELEASE_PACKAGE_LIBSQLITE3
Bug: 430889718
Bug: 281064726
Test: presubmit
Change-Id: I5f7183f2222638beb6a8067dead7f935b4848a11
---
 .../trunk_staging/RELEASE_PACKAGE_LIBSQLITE3.textproto        | 4 ----
 1 file changed, 4 deletions(-)
 delete mode 100644 flag_values/trunk_staging/RELEASE_PACKAGE_LIBSQLITE3.textproto

diff --git a/flag_values/trunk_staging/RELEASE_PACKAGE_LIBSQLITE3.textproto b/flag_values/trunk_staging/RELEASE_PACKAGE_LIBSQLITE3.textproto
deleted file mode 100644
index bb903b8d..00000000
--- a/flag_values/trunk_staging/RELEASE_PACKAGE_LIBSQLITE3.textproto
+++ /dev/null
@@ -1,4 +0,0 @@
-name: "RELEASE_PACKAGE_LIBSQLITE3"
-value: {
-  string_value: "3440300"
-}
-- 
2.51.2.1006.ga50a493c49-goog

